src/api/java/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
[This comment also applies to the `gradle-wrapper.jar`]

If I understand this correctly (I've never used gradle myself) these files have been generated by you using gradle.
Even though gradle describes this as a "one-time process", we should make sure that we can actually reproduce this step. I'd suggest to handle this similar to ANTLR3 (which also downloads the jar file on demand).